web-app/__mocks__/sweetalert2.js

```diff
@@ -0,0 +1 @@
+export default jest.fn(() => {});
```

Comment:
Is there any reason for moving these from `setupTests.js`?

Reply:
I was experimenting with mocks... Seems to be best practice to move all mocks into a __mocks__ folder